invenio_openaire/loaders.py

```diff
@@ -211,6 +215,46 @@ def grantxml2json(self, grant_xml):
         }
         return ret_json
 
+    def grantjson2json(self, grant_json):
+        """Convert OpenAIRE grant JSON into comptaible JSON."""
+        funder_doi = 0  # data is available, just have to be decided how it is fetched depending on the final integration
+        
+        to_replace = 0  # Code used to generate eurepo identifier
+
+        grant_id = grant_json["code"]
+        id_ = f"{funder_doi}::{grant_id}"
+
+        ret_json = {
+            "created": datetime.isoformat(datetime.now()),
+            "id": id_,
+            "links": {
+                "self": f"https://zenodo.org/api/grants/{id_}"
+            },
+            "metadata": {
+                "$schema": "http://zenodo.org/schemas/grants/grant-v1.0.0.json",
```

Comment:
```suggestion
                "$schema": self.schema_formatter.schema_url,
```

As done in the other loader, which will make life a bit easier for tests too.